src/compiler/commandLineParser.ts

```diff
@@ -200,6 +200,14 @@ namespace ts {
             description: Diagnostics.Enable_incremental_compilation,
             transpileOptionValue: undefined
         },
+        {
+            name: "noIndirectImports",

```

Comment:
`noIndirectImports` sounds like an option to disallow indirect imports—should it be something like `skipIndirectImportCheck` instead?

Reply:
`Check` part becomes ambiguous since from incremental point of view we are skipping type check of these files as well as not emitting d.ts files for the files that indirectly import the file with changed signature